Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "DocumenterCitations"
 uuid = "daee34ce-89f3-4625-b898-19384cb65244"
 authors = ["Michael Goerz <mail@michaelgoerz.net>"]
-version = "0.2.1"
+version = "0.3.0-dev"
```

Comment:
```suggestion
version = "1.0.0-dev"
```

Reply:
We can certainly upgrade to Documenter 1.0 in a non-breaking way. I think it is just mostly very annoying for users if we will have two breaking (according to the Pkg interpretation of SemVer) releases in quick succession.

Reply:
Good point. Alright, I'm good with going to 1.0, then (but I'd still prefer for someone to give this a try in the wild before pushing out the release)

Reply:
Ok, I changed any mention of 0.3.0 to 1.0, targeting a 1.0 release in the near future